File: main/scala/mod/iceandshadow3/compat/CRefItem.java
package mod.iceandshadow3.compat;

import net.minecraft.block.Block;
import net.minecraft.item.ItemStack;

//TODO: Manually generated class stub.
public class CRefItem {
	final ItemStack is;
	CRefItem(ItemStack itemstack) {
		is = itemstack;
	}
	public int count() {return is.getCount();}
	public int countMax() {return is.getMaxStackSize();}
}

File: main/scala/mod/iceandshadow3/compat/CRefItem.scala
package mod.iceandshadow3.compat

import net.minecraft.item.ItemStack

//TODO: Manually generated class stub.
class CRefItem(private[compat] val is: ItemStack) {
	def count(): Int = is.getCount();
	def countMax(): Int = is.getMaxStackSize();
}
